benchmarks/tests/test_csv_benchmark.py

```diff
@@ -17,25 +14,25 @@ def assert_benchmark(self, result, case, source, language):
         }
         if language == "R":
             expected["language"] = "R"
-        assert munged["tags"] == expected
-        _asserts.assert_info_and_context(munged, language=language)
+        assert result.tags == expected
+        _asserts.assert_info_and_context(result, language=language)
 
     def assert_run_py(self, run, index, case, source):
         if "data_frame" not in case:
-            result, output = run[index]
+            result = run[index]
             self.assert_benchmark(
                 result=result, case=case, source=source.name, language="Python"
             )
             if self.benchmark.name == "csv-read":
-                _asserts.assert_table_output(source.name, output)
+                _asserts.assert_table_output(source.name, result.output)
 
     def assert_run_r(self, run, index, case, source):
         if "streaming" not in case:
-            result, output = run[index]
```

Comment:
Conbench expects to have result and output to be returned by benchmark.run(): https://github.com/conbench/conbench/blob/main/conbench/cli.py#L149